arm-apimanagement/2014-02-14/swagger/service.json

```diff
@@ -0,0 +1,1289 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "APIManagementServiceClient",
+	"description": "REST API for Azure APIManagement Service",
```

Comment:
Could you format this JSON file? It's good practice to have things like spacing consistent.